_topic_maps/_topic_map.yml

```diff
@@ -95,9 +95,18 @@ Topics:
 - Name: Selecting an installation method and preparing a cluster
   File: installing-preparing
   Distros: openshift-origin,openshift-enterprise
-- Name: Mirroring images for a disconnected installation
-  File: installing-mirroring-installation-images
+- Name: Disconnected installation mirroring
+  Dir: disconnected_install
   Distros: openshift-origin,openshift-enterprise
+  Topics:
+  - Name: About disconnected installation mirroring
+    File: index
+  - Name: Creating a mirror registry
+    File: installing-mirroring-creating-registry
+  - Name: Mirroring images for a disconnected installation
+    File: installing-mirroring-installation-images
+  - Name: Mirroring images for a disconnected installation using the oc-mirror plug-in
```

Comment:
@afflom @jpower432 Are the oc-mirror docs appropriate for OKD too, or should it just be for OCP?

Reply:
Yes, currently the steps for the user would be the same for mirroring OCP or OKD.